app/src/main/java/com/github/polypoly/app/MapActivity.kt

```diff
@@ -59,6 +63,12 @@ class MapActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val db = Firebase.database
+        try {
+            db.setPersistenceEnabled(false)
+        } catch(_: java.lang.Exception) { }
+        GlobalInstances.remoteDB = RemoteDB(db, "live")
+
```

Comment:
Not sure to understand what this is doing here. API specific code that isn't too self explicit (i.e I am assuming this initializes the database or something but it might be more fitting to encapsulate it in a function called "initDB" for example): db.setPersistence has no meaning to someone who has not read FireBase Doc or who has no DB knowledge maybe!

Reply:
It provides better consistency between the local firebase database cache and the remote one. I added a comment to make it clearer for now, this might actually become useless if we manually cache data in a further sprint.